osf/migrations/0138_ensure_subjects_and_providers.py

```diff
@@ -85,22 +85,22 @@ def create_subjects(state, schema):
                     subject.save()
 
 def create_osf_registries(state, schema):
-    AbstractProvider = state.get_model('osf.abstractprovider')
-    if not AbstractProvider.objects.filter(_id=OSF_REGISTRIES_DATA['_id'], type=OSF_REGISTRIES_DATA['type']).exists():
+    RegistrationProvider = state.get_model('osf.registrationprovider')
+    if not RegistrationProvider.objects.filter(_id=OSF_REGISTRIES_DATA['_id'], type=OSF_REGISTRIES_DATA['type']).exists():
         logger.info('Creating RegistrationProvider "osf"')
-        _create_provider(AbstractProvider, OSF_REGISTRIES_DATA, state)
+        _create_provider(RegistrationProvider, OSF_REGISTRIES_DATA, state)
 
-def noop(*args, **kwargs):
-    pass
 
 class Migration(migrations.Migration):
 
     dependencies = [
         ('osf', '0137_auto_20181012_1756'),
     ]
 
-    operations = [] if getattr(settings, 'TEST_ENV', False) else [
-        migrations.RunPython(create_osf_preprints, noop),
-        migrations.RunPython(create_subjects, noop),
-        migrations.RunPython(create_osf_registries, noop),
+    operations = [
+        migrations.RunPython(create_osf_registries, migrations.RunPython.noop),
+    ] if getattr(settings, 'TEST_ENV', False) else [
```

Comment:
I know you had to jump through a bunch of hoops to get this working, but I just want to verify that everything creates properly in a fresh local database without the creating of subjects or preprints.

Reply:
Yes, it works within normal testing using pytest.